### DISCUSSION TITLE
tgcf 0.1.17

### DISCUSSION THREAD

**Discussion by aahnik** [aahnik/tgcf#58] [General]
I published a new release!

- [x] refactor code for better maintainability
- [x] the features of paid private repo "telegram-live-sync" is now free and merged into tgcf
- [x] implement plugin architecture to allow easy addition of features
- [x] a shiny new README

<hr /><em>This discussion was created from the release <a href='https://github.com/aahnik/tgcf/releases/tag/v0.1.17'>tgcf 0.1.17</a>.</em>

**Comment by u-8d** [comment]
Kudos🤘

**Comment by Abhishek0863** [comment]
Amazing, i love it 😍

& Thanks for this release

**Reply by aahnik** [reply]
wait for a few days. i need to complete writing the documentation make the videos.

in the mean time, help tgcf reach 100 stars, by sharing with your friends

**Comment by u-8d** [comment]
![Screenshot_20210423_135459](https://user-images.githubusercontent.com/81148580/115842779-c92b0e80-a43b-11eb-99a8-5b7843a70b28.jpg)

Bro unable to Y or n in termux

**Reply by aahnik** [reply]
you are by mistake clicking some other key. the process is getting aborted.

see your screenshot. it is showing "Abort". you may just hit 'y'. i think you are trying to make it uppercase, and that key is aborting the process.

see the latest guide here : https://github.com/aahnik/tgcf/wiki/Run-on-Android-using-Termux

**Comment by ghost** [comment]
amazing.
